packages/terra-demographics-banner/src/selectable-demographics-banner/SelectableDemographicsBanner.jsx

```diff
@@ -0,0 +1,170 @@
+  /**
+   * The column layout in which to present the displays.
+   */
+  postMenstrualAge: PropTypes.string,
+  /**
+   * The persons preferred first name if they have one.
+   */
+  preferredFirstName: PropTypes.string,
+  /**
+   * Callback function triggered when info button is clicked.
+   */
+  infoButtonOnClick: PropTypes.func.isRequired,
+  /**
+   * Callback function triggered when selectable-tile is clicked.
+   */
+  selectableTileOnClick: PropTypes.func.isRequired,
+  /**
+   * Id for Info Button.
+   */
+  infoButtonId: PropTypes.string,
+  /**
+  * Id for Selectable Tile.
+  */
+  selectableTileId: PropTypes.string,
+};
+
+const defaultProps = {
+  applicationContent: null,
+  age: undefined,
+  dateOfBirth: undefined,
+  deceasedDate: null,
+  gender: undefined,
+  gestationalAge: null,
+  identifiers: {},
+  personName: undefined,
+  photo: undefined,
+  postMenstrualAge: null,
+  preferredFirstName: null,
+  infoButtonId: undefined,
+  selectableTileId: undefined,
+};
+
+const SelectableDemographicsBanner = ({
+  age,
+  dateOfBirth,
+  gender,
+  intl,
+  personName,
+  infoButtonOnClick,
+  selectableTileOnClick,
+  infoButtonId,
+  selectableTileId,
+  ...customProps
+}) => {
+  const noDataProvided = intl.formatMessage({ id: 'Terra.demographicsBanner.noDataProvided' });
+
+  const handleKeyDown = (event) => {
+    console.log('keyspace', KEY_SPACE);
+    console.log('keyspace', KEY_RETURN);
+    if (event.keyCode === KEY_SPACE || event.keyCode === KEY_RETURN) {
+      event.preventDefault();
+    }
+  };
+
+  const handleKeyUp = (event) => {
+    if ((event.keyCode === KEY_SPACE || event.keyCode === KEY_RETURN) && selectableTileOnClick) {
+      selectableTileOnClick();
+    }
+  };
+
+  const handleOnClick = () => {
+    if (selectableTileOnClick) {
+      selectableTileOnClick();
+    }
+  };
+
+  const DemographicsBannerSelectableTile = (
+    <DemographicsBannerDisplay
+      {...customProps}
+      className={cx('selectable-demographics-banner')}
+      age={age || noDataProvided}
+      dateOfBirth={dateOfBirth || noDataProvided}
+      gender={gender || noDataProvided}
+      personName={personName || noDataProvided}
+      dateOfBirthLabel={intl.formatMessage({ id: 'Terra.demographicsBanner.dateOfBirth' })}
+      dateOfBirthFullText={intl.formatMessage({ id: 'Terra.demographicsBanner.dateOfBirthFullText' })}
+      deceasedDateLabel={intl.formatMessage({ id: 'Terra.demographicsBanner.deceased' })}
+      gestationalAgeLabel={intl.formatMessage({ id: 'Terra.demographicsBanner.gestationalAge' })}
+      gestationalAgeFullText={intl.formatMessage({ id: 'Terra.demographicsBanner.gestationalAgeFullText' })}
+      postMenstrualAgeLabel={intl.formatMessage({ id: 'Terra.demographicsBanner.postMenstrualAge' })}
+      postMenstrualAgeFullText={intl.formatMessage({ id: 'Terra.demographicsBanner.postMenstrualAgeFullText' })}
+    />
+  );
+
+  return (
+    <BackgroundTile deceasedDate={(customProps.deceasedDate)}>
```

Comment:
deceasedDate is a first class prop, it should be spread above and not keyed by the customProps spread

Reply:
added [new prop](https://github.com/cerner/terra-core/blob/b7897458163ba5b8e2bd88cfc5faa93b14d56aff/packages/terra-demographics-banner/src/SelectableDemographicsBanner.jsx#L156) for deceased state